src/renderer/navigation/page-param.ts

```diff
@@ -0,0 +1,132 @@
+// Manage observable URL-param from document.location.search
+import { IObservableHistory } from "mobx-observable-history";
+
+export interface PageParamInit<V = any> {
+  name: string;
+  isSystem?: boolean;
```

Comment:
This shouldn't be exposed in the extensions api

Reply:
Why not? this interface is used in exporting helper `Navigation.createPageParam()`

Reply:
Yes, third-party devs shouldn't see this flag

Reply:
Fixed